snapshot/lib/snapshot/simulator_launchers/simulator_launcher.rb

```diff
@@ -0,0 +1,74 @@
+require 'snapshot/simulator_launchers/simulator_launcher_base'
+
+module Snapshot
+  class SimulatorLauncher < SimulatorLauncherBase
+    def default_number_of_simultaneous_simulators
+      cpu_count = OS.cpu_count
```

Comment:
I agree, we haven't linked it up yet, but that's good to call out. We did some testing and having too many sims causes serious perf issues.